schemas.md

```diff
@@ -91,87 +91,56 @@ The FOIA metadata file should instruct the Platform:
 - what additional fields are required for a perfected request
 - what additional fields the agency would like to collect
 - explanation of additional requirements†
-- FOIA contact information
 
 _† Our research has shown that users often find additional requirements for FOIA
 submission confusing and burdensome. We hope that by having agencies include
 supplemental information about the requirements, it will help inform requesters
 and allay any concerns._
 
-* submission format
-* required form fields
-  * name
-  * label
-  * URL to the regulations requiring additional information
-  * help text
-* additional form fields
-  * name
-  * label
-  * help text
-* uiSchema describing form presentation
-* reading room URL
-* FOIA contact information
-  * name
-  * title
-  * address
-  * phone
-  * fax
-  * email
-  * website
-  * notes
-  * FOIA service center
-    * name
-    * phone
-  * FOIA public liason
-    * name
-    * phone
-
-FOIA Contact information is meant to include what already exists on foia.gov.
-
+The metadata file should include the following fields:
 
-### Submission format
-
-In order to provide flexibility to agencies accepting submissions from the
-National FOIA Platform, there are two types of submission formats. HTTPS API and
-by email. Agencies should only accept requests directly from the National FOIA
-Platform.
-
-
-#### HTTPS API
+* agency abbreviation
+* list of departments
+  * name
+  * list of submission methods
+    * submission format
 
-For agencies that can provide an HTTPS API, the API should accept a `POST` request
-containing the [request schema](#creating-a-request) above.
+You can see [example metadata files
+here](https://github.com/18F/foia-recommendations/tree/master/examples).
 
-The metadata file should contain this additional data as part of the submission
-format:
+The schema for this metadata file is partially inspired by the [2015 FOIA
+work](https://github.com/18F/2015-foia/tree/master/contacts/data) around
+collecting contact information.
 
-* format: `https`
-* url
 
+### Submission methods
 
-#### Email
+A list of methods that the agency supports for receiving a FOIA request. Each
+method represents a different format. Example formats include `usps`, `fax`,
+`email`, `web`, and `platform`.
 
-For agencies that cannot yet provide an HTTPS API, email is a familiar
-submission mechanism. The email body will contain the submission body with any
-uploaded files as email attachments. Agencies can specify additional sub-formats
-that may be delivered by email, `text`, `headers`, `csv`.
 
-The metadata file should contain this additional data as part of the submission
-format:
+#### Platform
 
-* format: `email`
-* email submission address
+If the agency wants to receive requests through the National FOIA Platform, they
+can include a `submission_method` with the `submission_format: platform`.
```

Comment:
No existing examples that I know of, though (a) we haven't talked to all the custom solution agencies, and (b) it doesn't hurt to be a little aspirational in specs like these, since sometimes it requires the articulation of an option for others to consider making it real.